custom-recipes/neo4j-export-nodes/recipe.json

```
@@ -148,6 +148,27 @@
             "description": "Nodes with this label will be deleted before run",
             "defaultValue": false,
             "visibilityCondition": "model.expert_mode"
+        },
+        {
+            "label": "Handling NULL Values",
+            "type": "SEPARATOR",
+            "visibilityCondition": "model.expert_mode"
+        },
+        {
+            "name": "na_values",
+            "label": "Custom NULL Values",
+            "type": "STRINGS",
+            "description": "Specify custom string values to be interpreted as NULL prior to the export to Neo4j. No values supplied results in no NULL values, thus empty cells will be replaced with the string 'null'",
```

Let's keep the description in the UI really simple and just copy what pandas did
```suggestion
            "description": "Additional strings to recognize as NaN values prior to the export to Neo4j",
```